yellowbrick/classifier/classification_report.py

```diff
@@ -4,6 +4,7 @@
 # Author:   Rebecca Bilbro <rbilbro@districtdatalabs.com>
 # Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
 # Author:   Neal Humphrey
+# Author:   Allyssa Riley
```

Comment:
@lwgray you could add your name here as well!

Reply:
sure!